README.md

```diff
@@ -1,7 +1,29 @@
 # advocates
 
-Cosmos Advocates
+The Cosmos Advocates repo tracks doc issues, feature ideas, and other collaborative doc efforts. The Cosmos Advocates people care about developer onboarding of Cosmos technology, documentation efforts, and cross-team coordination. 
 
-## Docs list
+## Reporting doc issues 
 
-See the working [list of docs](./docs_list.md/) for the primary owners and links to the published documentation and the repo or repos.
+For general outstanding tasks for docs, create an issue here in cosmos/advocates repo. 
+
+When you identify specific technical errors, create a doc issue in that project repository. See the working [list of docs](./docs_list.md/) for the primary content owners and links to the published documentation and the repo or repos. Technical code maintainers for projects are mandated to update documentation, like [Cosmos SDK](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#updating-documentation). 
+
+For all issues, help us out. Provide helpful details of the doc that requires attention, including:
+
+- topic
+- links
+- context
+- version
+- screen shots
+- other relevant information
+
+## Issue triage 
+
+As we scale our content teams, issue triage is a fluid process that we're still working out. Thank you for your patience. Thank you for caring.
+
+- For Cosmos SDK and Starport docs, tag @barriebyron 
+- For Cosmos SDK tutorials, tag @toschdev 
```

Comment:
some things not covered that should have owners:
- IBC protocol website https://ibcprotocol.org/
- rust IBC & hermes relayer (should be andy)
- typescript relayer
- tendermint core
- tendermint spec repo
- IBC spec repo
- cosmjs